shared/actions/login/index.js

```diff
@@ -343,6 +350,21 @@ export function logoutDone () : AsyncAction {
   }
 }
 
+export function clearStoredSecret (username: string) : AsyncAction {
+  return (dispatch, getState) => {
+    const params: login_clearStoredSecret_rpc = {
+      ...makeWaitingHandler(dispatch),
```

Comment:
I was on the fence. Since this RPC is acting locally, it probably won't require a delay that is significant to the user. I'll take it out.